pkg/core/native/native_neo.go

```diff
@@ -1051,6 +1055,10 @@ func (n *NEO) getCandidatesCall(ic *interop.Context, _ []stackitem.Item) stackit
 	return stackitem.NewArray(arr)
 }
 
+func (n *NEO) GetCommitteeAddressCall(ic *interop.Context, _ []stackitem.Item) stackitem.Item {
```

Comment:
This function should not be exported.

s/`GetCommitteeAddressCall`/`getCommitteeAddress` to match the existing contract structure.